criu/memfd.c

```diff
@@ -339,6 +341,21 @@ int memfd_open(struct file_desc *d, u32 *fdflags)
 
 	/* Reopen the fd with original permissions */
 	flags = fdflags ? *fdflags : mfe->flags;
+
+	if (!mfi->inode->was_opened_rw && (flags & O_ACCMODE) == O_RDWR) {
+		/* HACK WARNING *
```

Comment:
DONT PANIC!

pls remove this warning and add an explanation why we are doing this.